core/src/js/services/mercenaries/highlights/mercenaries-synergies-highlight-processor.ts

```diff
@@ -700,6 +713,17 @@ const frost = (card: ReferenceCard) => spellSchool(card, SpellSchool.FROST);
 const fel = (card: ReferenceCard) => spellSchool(card, SpellSchool.FEL);
 const shadow = (card: ReferenceCard) => spellSchool(card, SpellSchool.SHADOW);
 
+const hasTag = (card: ReferenceCard, tag: GameTag) => (card?.referencedTags ?? []).includes(GameTag[tag]);
```

Comment:
Based on the above comment, I would rename this to hasReferencedTag

Reply:
Done